src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/Scheduler/config.yml

```diff
@@ -10,6 +10,9 @@ services:
     Symfony\Bundle\FrameworkBundle\Tests\Fixtures\Messenger\DummySchedule:
         autoconfigure: true
 
+    Symfony\Bundle\FrameworkBundle\Tests\Fixtures\Messenger\DummyTask:
+        autoconfigure: true
```

Comment:
There are no tests associated with this, can you add some @valtzu?
I've moved these to a new schedule to avoid breaking the existing tests (https://github.com/symfony/symfony/pull/51802).

Reply:
@fabpot I'm not too familiar with the framework tests yet, and I'm also a bit busy now. I would need a week or two, if that's ok then I can try.

Reply:
Sure, there is no rush. Ping me if you need help.